NDB.Covid19/NDB.Covid19.iOS/Views/FarewellSmittestopPage/FarewellSmittestopPageViewController.cs

```diff
@@ -38,6 +40,32 @@ public void SetupStyling()
 
             TextThreeFarwellLbl.Font = StyleUtil.Font(StyleUtil.FontType.FontRegular, 16, 22);
             TextThreeFarwellLbl.Text = FarewellSmittestopViewModel.FAREWELL_SMITTESTOP_BODY_THREE;
+
+            SetupMoreInfo();
+        }
+
+        private void SetupMoreInfo()
+        {
+            StyleUtil.InitLabelWithSpacing(MoreInfoLabel, StyleUtil.FontType.FontRegular,
+                FarewellSmittestopViewModel.SMITTESTOP_NOT_ACTIVE_MORE_INFO, 1.24, 18, 24, UITextAlignment.Center);
```

Comment:
Look like this string is missing from the view model?

Reply:
Sorry, forgot to add file in commit. Added now :-)